src/controllers/accommodation.controller.js

```diff
@@ -0,0 +1,21 @@
+import AccommodationService from '../services/accomodation.service';
+import response from '../helpers/response.helper';
+
+/**
+ * This class contains functions for all accommodation controller.
+ * @class NotificationController
+ */
+class AccommodationController {
+  /**
+   * Like an accommodation
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Promise} res
+   */
+  static async likeOrUnlike(req, res) {
+    const query = await AccommodationService.likeOrUnlike(req.body.isLike, req.user.id, req.params.accommodationId);
+    const likeOrUnlike = req.body.isLike ? 'Like' : 'Unlike';
+    return response.successMessage(res, `${likeOrUnlike} was successfully processed`, 200, query);
```

Comment:
@VedaDom, I do not think we should be returning this message, may you find a more user friendly message?